### PR TITLE
Add feature page for Custom Filters in Envoy

### PR DESCRIPTION
**Pull request by johnma14**

### CODE REVIEW

**Comment by brian-avery** [comment]
@istio/wg-networking-maintainers can you please review?

**Comment by johnma14** [comment]
@howardjohn @istio/wg-networking-maintainers can we get a review on this one. Thank you.

File: features/envoy-custom-filters.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+- [ ] Latency, throughput, and scalability are quantified and documented on
+	istio.io. 
+
+**Bugs**
+
+- [x] Feature has no known major issues. 
+
+**Approvals**
+
+- [x] The appropriate work group(s) have reviewed and approved promotion of the feature.
```
Comment (howardjohn) [inline features/envoy-custom-filters.md:207]: This has not been approved for stable.

**Comment by johnma14** [comment]
@howardjohn you are right. It is an alpha feature. I do not have any more info about this feature in terms of design docs, etc but I will take a look. Do we have tests for this feature?

**Pull request closed without merging**